src/glue_launcher_lambda/glue_launcher.py

```diff
@@ -0,0 +1,517 @@
+
+    # Parse command line inputs and set defaults
+    parser.add_argument("--aws-profile", default="default")
+    parser.add_argument("--aws-region", default="eu-west-2")
+    parser.add_argument("--environment", help="Environment value", default="UNSET_TEXT")
+    parser.add_argument("--application", help="Application", default="UNSET_TEXT")
+    parser.add_argument("--log-level", help="Log level for lambda", default="INFO")
+
+    _args = parser.parse_args()
+
+    # Override arguments with environment variables where set
+    if "AWS_PROFILE" in os.environ:
+        _args.aws_profile = os.environ["AWS_PROFILE"]
+
+    if "AWS_REGION" in os.environ:
+        _args.aws_region = os.environ["AWS_REGION"]
+
+    if "ENVIRONMENT" in os.environ:
+        _args.environment = os.environ["ENVIRONMENT"]
+
+    if "APPLICATION" in os.environ:
+        _args.application = os.environ["APPLICATION"]
+
+    if "LOG_LEVEL" in os.environ:
+        _args.log_level = os.environ["LOG_LEVEL"]
+
+    if "JOB_QUEUE_DEPENDENCIES" in os.environ:
+        dependencies = os.environ["JOB_QUEUE_DEPENDENCIES"].split(",")
+        _args.job_queue_dependencies = dependencies
+
+    if "ATHENA_S3_OUTPUT_LOCATION" in os.environ:
+        _args.athena_s3_output_location = os.environ["ATHENA_S3_OUTPUT_LOCATION"]
+
+    if "MISSING_IMPORTS_TABLE_NAME" in os.environ:
+        _args.missing_imports_table_name = os.environ["MISSING_IMPORTS_TABLE_NAME"]
+
+    if "MISSING_EXPORTS_TABLE_NAME" in os.environ:
+        _args.missing_exports_table_name = os.environ["MISSING_EXPORTS_TABLE_NAME"]
+
+    if "COUNTS_TABLE_NAME" in os.environ:
+        _args.counts_table_name = os.environ["COUNTS_TABLE_NAME"]
+
+    if "MISMATCHED_TIMESTAMPS_TABLE_NAME" in os.environ:
+        _args.mismatched_timestamps_table_name = os.environ[
+            "MISMATCHED_TIMESTAMPS_TABLE_NAME"
+        ]
+
+    if "ETL_GLUE_JOB_NAME" in os.environ:
+        _args.etl_glue_job_name = os.environ["ETL_GLUE_JOB_NAME"]
+
+    if "MANIFEST_S3_INPUT_LOCATION_IMPORT_HISTORIC" in os.environ:
+        _args.manifest_s3_input_location_import_historic = os.environ[
+            "MANIFEST_S3_INPUT_LOCATION_IMPORT_HISTORIC"
+        ]
+
+    if "MANIFEST_S3_INPUT_LOCATION_EXPORT_HISTORIC" in os.environ:
+        _args.manifest_s3_input_location_export_historic = os.environ[
+            "MANIFEST_S3_INPUT_LOCATION_EXPORT_HISTORIC"
+        ]
+
+    if "MANIFEST_COMPARISON_CUT_OFF_DATE_START" in os.environ:
+        _args.manifest_comparison_cut_off_date_start = os.environ[
+            "MANIFEST_COMPARISON_CUT_OFF_DATE_START"
+        ]
+    else:
+        _args.manifest_comparison_cut_off_date_start = "1983-11-15T09:09:55.000"
+
+    if "MANIFEST_COMPARISON_CUT_OFF_DATE_END" in os.environ:
+        _args.manifest_comparison_cut_off_date_end = os.environ[
+            "MANIFEST_COMPARISON_CUT_OFF_DATE_END"
+        ]
+    else:
+        _args.manifest_comparison_cut_off_date_end = "2099-11-15T09:09:55.000"
```

Comment:
I'm not sure how you will be able to pass this and the above in from the terraform as they need to be relevant to the time running. I would set the default here to the default that is needed when run. Concourse usually sets them in bash and passes them at runtime, but that isn't possible here, so they need to be set at runtime here.

Cut off date end should be midnight of current day and cut off date start should default to midnight of yesterday I think. Worth checking the manifest comparison concourse code to be 100% sure.